src/assets/js/pages/NewEventPage/components/NewEventForm.js

```diff
@@ -67,7 +67,9 @@ class NewEventForm extends Component {
 
                 {fields.createColumn('col-sm-12 col-md-8 text-right',
                     <button className={'btn sd-form__nav-button rounded-button ' +
-                        'success button'} type="submit" disabled={pristine || submitting}>Apply!</button>
+                        'success button'} type="submit" disabled={pristine || submitting}>
+                        Create Event!
+                    </button>
                 )}
             </form>
         )
```

Comment:
This entire file is 4-space indented rather than 2.

Reply:
Also the `const {pristine, submitting, handleSubmit} = this.props;` have not been added to the propTypes.